keep-ui/features/workflows/builder/ui/BuilderChat/AddTriggerUI.tsx

```
@@ -0,0 +1,217 @@
+  // TODO: validate triggerProperties here or in addNodeBetween??
+  const triggerDefinition = {
+    ...triggerTemplate,
+    properties: {
+      ...triggerTemplate.properties,
+      ...JSON.parse(triggerProperties),
+    },
+  };
+  return V2StepTriggerSchema.parse(triggerDefinition);
+}
+
+type SuggestionStatus = "complete" | "error" | "declined";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'SuggestionStatus'. Consider to delete it or rename it.

'SuggestionStatus' is defined here:


(lint/suspicious/noRedeclare)